app/views/chat_channels/index.html.erb

```diff
@@ -14,7 +14,8 @@
       data-pusher-key="<%= ApplicationConfig["PUSHER_KEY"] %>"
       data-github-token="<%= @github_token %>"
       data-chat-channels="<%= [] %>"
-      data-chat-options="<%= { showChannelsList: true, showTimestamp: true, activeChannelId: @active_channel&.id, currentUserId: current_user.id }.to_json %>">
+      data-chat-options="<%= { showChannelsList: true, showTimestamp: true, activeChannelId: @active_channel&.id, currentUserId: current_user.id }.to_json %>"
+      data-tag-moderator=<%= { isTagModerator: current_user.has_role?(:tag_moderator) }.to_json %>>
```

Comment:
This seems to be incorrect which is also why @nickytonline can't see the + button despite making his user a tag moderator locally.

```suggestion
      data-tag-moderator=<%= { isTagModerator: current_user.tag_moderator?) }.to_json %>>
```

I used a Rails console to verify this:

```ruby
user.has_role?(:tag_moderator)
#=> false
user.tag_moderator?
#=> true
```

If you look into `user.rb`, you'll see that the method is defined as follows:

```ruby
roles.where(name: "tag_moderator").any?
```